### ISSUE TITLE
Test Failure: Pods created by ReplicationController should spread to different node

### ISSUE THREAD

**Issue by monopole** [kubernetes/kubernetes#47382]
[jenkins](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-gke-cvm-1-6-gci-1-7-upgrade-cluster-new/6#k8sio-schedulerpriorities-serial-pods-created-by-replicationcontroller-should-spread-to-different-node)

[test-grid](https://k8s-testgrid.appspot.com/1.6-1.7-upgrade#gke-cvm-gci-upgrade-cluster-new&include-filter-by-regex=%5EOverall%24%7C%5C%5Bk8s%5C.io%5C%5D%5C%20SchedulerPriorities%5C%20%5C%5BSerial%5C%5D%5C%20Pods%5C%20created%5C%20by%5C%20ReplicationController%5C%20should%5C%20spread%5C%20to%5C%20different%5C%20node)

```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/scheduling/priorities.go:154
Jun 12 20:54:43.472: Pods are not spread to each node.
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/scheduling/priorities.go:152
```

Chronically failing test.  Perhaps covers deprecated behavior?

/sig scalability 
/assign @cheftako

**Comment by monopole** [comment]
shows up in [many old failure reports](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20%22Pods%20created%20by%20ReplicationController%20should%20spread%20to%20different%20node%22)

**Comment by k82cn** [comment]
/sig scheduling

`scheduling/priorities.go` is one of scheduling e2e test.

**Comment by k82cn** [comment]
/assign 

I'll check this one.

**Comment by k82cn** [comment]
According to the log, the balanced pods were not created by scheduler, the cpu/mem usage was unchanged.

```
�[1mSTEP�[0m: Create a pod for each node to make the nodes have almost same cpu/mem usage ratio 
Jun 12 20:54:33.154: INFO: ComputeCpuMemFraction for node: gke-bootstrap-e2e-default-pool-5334c705-kvjn 
Jun 12 20:54:33.160: INFO: Pod for on the node: fluentd-gcp-v2.0-smzn0, Cpu: 200, Mem: 419430400 
Jun 12 20:54:33.160: INFO: Pod for on the node: ip-masq-agent-fqqhz, Cpu: 10, Mem: 8388608 
Jun 12 20:54:33.160: INFO: Pod for on the node: kube-proxy-gke-bootstrap-e2e-default-pool-5334c705-kvjn, Cpu: 100, Mem: 209715200 
Jun 12 20:54:33.161: INFO: Node: gke-bootstrap-e2e-default-pool-5334c705-kvjn, totalRequestedCpuResource: 410, cpuAllocatableMil: 2000, cpuFraction: 0.205 
Jun 12 20:54:33.161: INFO: Node: gke-bootstrap-e2e-default-pool-5334c705-kvjn, totalRequestedMemResource: 742391808, memAllocatableVal: 7848955904, memFraction: 0.09458478517144693 
Jun 12 20:54:33.161: INFO: ComputeCpuMemFraction for node: gke-bootstrap-e2e-default-pool-5334c705-m9p0 
Jun 12 20:54:33.167: INFO: Pod for on the node: event-exporter-v0.1.0-2428992396-051pw, Cpu: 200, Mem: 419430400 
Jun 12 20:54:33.167: INFO: Pod for on the node: fluentd-gcp-v2.0-8sdsm, Cpu: 200, Mem: 419430400 
Jun 12 20:54:33.167: INFO: Pod for on the node: heapster-v1.3.0-1150159031-mzqdk, Cpu: 138, Mem: 309100544 
Jun 12 20:54:33.167: INFO: Pod for on the node: ip-masq-agent-7pkl3, Cpu: 10, Mem: 8388608 
Jun 12 20:54:33.167: INFO: Pod for on the node: kube-dns-1148326581-m735b, Cpu: 260, Mem: 115343360 
Jun 12 20:54:33.167: INFO: Pod for on the node: kube-dns-autoscaler-3203507111-t26lm, Cpu: 20, Mem: 10485760 
Jun 12 20:54:33.169: INFO: Pod for on the node: kube-proxy-gke-bootstrap-e2e-default-pool-5334c705-m9p0, Cpu: 100, Mem: 209715200 
Jun 12 20:54:33.169: INFO: Pod for on the node: l7-default-backend-1044750973-vh9sr, Cpu: 10, Mem: 20971520 
Jun 12 20:54:33.169: INFO: Node: gke-bootstrap-e2e-default-pool-5334c705-m9p0, totalRequestedCpuResource: 1038, cpuAllocatableMil: 2000, cpuFraction: 0.519 
Jun 12 20:54:33.169: INFO: Node: gke-bootstrap-e2e-default-pool-5334c705-m9p0, totalRequestedMemResource: 1617723392, memAllocatableVal: 7848955904, memFraction: 0.20610682640930275 
Jun 12 20:54:33.169: INFO: ComputeCpuMemFraction for node: gke-bootstrap-e2e-default-pool-5334c705-nb00 
Jun 12 20:54:33.175: INFO: Pod for on the node: fluentd-gcp-v2.0-k7c5j, Cpu: 200, Mem: 419430400 
Jun 12 20:54:33.175: INFO: Pod for on the node: ip-masq-agent-b4dfl, Cpu: 10, Mem: 8388608 
Jun 12 20:54:33.175: INFO: Pod for on the node: kube-dns-1148326581-ngxn6, Cpu: 260, Mem: 115343360 
Jun 12 20:54:33.175: INFO: Pod for on the node: kube-proxy-gke-bootstrap-e2e-default-pool-5334c705-nb00, Cpu: 100, Mem: 209715200 
Jun 12 20:54:33.175: INFO: Pod for on the node: kubernetes-dashboard-1971317102-nzgtf, Cpu: 100, Mem: 104857600 
Jun 12 20:54:33.175: INFO: Node: gke-bootstrap-e2e-default-pool-5334c705-nb00, totalRequestedCpuResource: 770, cpuAllocatableMil: 2000, cpuFraction: 0.385 
Jun 12 20:54:33.175: INFO: Node: gke-bootstrap-e2e-default-pool-5334c705-nb00, totalRequestedMemResource: 962592768, memAllocatableVal: 7848955904, memFraction: 0.12263959433246932 

�[1mSTEP�[0m: Compute Cpu, Mem Fraction after create balanced pods. 
Jun 12 20:54:33.209: INFO: ComputeCpuMemFraction for node: gke-bootstrap-e2e-default-pool-5334c705-kvjn 
Jun 12 20:54:33.214: INFO: Pod for on the node: fluentd-gcp-v2.0-smzn0, Cpu: 200, Mem: 419430400 
Jun 12 20:54:33.214: INFO: Pod for on the node: ip-masq-agent-fqqhz, Cpu: 10, Mem: 8388608 
Jun 12 20:54:33.214: INFO: Pod for on the node: kube-proxy-gke-bootstrap-e2e-default-pool-5334c705-kvjn, Cpu: 100, Mem: 209715200 
Jun 12 20:54:33.214: INFO: Node: gke-bootstrap-e2e-default-pool-5334c705-kvjn, totalRequestedCpuResource: 410, cpuAllocatableMil: 2000, cpuFraction: 0.205 
Jun 12 20:54:33.214: INFO: Node: gke-bootstrap-e2e-default-pool-5334c705-kvjn, totalRequestedMemResource: 742391808, memAllocatableVal: 7848955904, memFraction: 0.09458478517144693 

�[1mSTEP�[0m: Compute Cpu, Mem Fraction after create balanced pods. 
Jun 12 20:54:33.214: INFO: ComputeCpuMemFraction for node: gke-bootstrap-e2e-default-pool-5334c705-m9p0 
Jun 12 20:54:33.223: INFO: Pod for on the node: event-exporter-v0.1.0-2428992396-051pw, Cpu: 200, Mem: 419430400 
Jun 12 20:54:33.223: INFO: Pod for on the node: fluentd-gcp-v2.0-8sdsm, Cpu: 200, Mem: 419430400 
Jun 12 20:54:33.223: INFO: Pod for on the node: heapster-v1.3.0-1150159031-mzqdk, Cpu: 138, Mem: 309100544 
Jun 12 20:54:33.223: INFO: Pod for on the node: ip-masq-agent-7pkl3, Cpu: 10, Mem: 8388608 
Jun 12 20:54:33.223: INFO: Pod for on the node: kube-dns-1148326581-m735b, Cpu: 260, Mem: 115343360 
Jun 12 20:54:33.223: INFO: Pod for on the node: kube-dns-autoscaler-3203507111-t26lm, Cpu: 20, Mem: 10485760 
Jun 12 20:54:33.223: INFO: Pod for on the node: kube-proxy-gke-bootstrap-e2e-default-pool-5334c705-m9p0, Cpu: 100, Mem: 209715200 
Jun 12 20:54:33.223: INFO: Pod for on the node: l7-default-backend-1044750973-vh9sr, Cpu: 10, Mem: 20971520 
Jun 12 20:54:33.224: INFO: Node: gke-bootstrap-e2e-default-pool-5334c705-m9p0, totalRequestedCpuResource: 1038, cpuAllocatableMil: 2000, cpuFraction: 0.519 
Jun 12 20:54:33.224: INFO: Node: gke-bootstrap-e2e-default-pool-5334c705-m9p0, totalRequestedMemResource: 1617723392, memAllocatableVal: 7848955904, memFraction: 0.20610682640930275 

�[1mSTEP�[0m: Compute Cpu, Mem Fraction after create balanced pods. 
Jun 12 20:54:33.224: INFO: ComputeCpuMemFraction for node: gke-bootstrap-e2e-default-pool-5334c705-nb00 
Jun 12 20:54:33.230: INFO: Pod for on the node: fluentd-gcp-v2.0-k7c5j, Cpu: 200, Mem: 419430400 
Jun 12 20:54:33.230: INFO: Pod for on the node: ip-masq-agent-b4dfl, Cpu: 10, Mem: 8388608 
Jun 12 20:54:33.230: INFO: Pod for on the node: kube-dns-1148326581-ngxn6, Cpu: 260, Mem: 115343360 
Jun 12 20:54:33.230: INFO: Pod for on the node: kube-proxy-gke-bootstrap-e2e-default-pool-5334c705-nb00, Cpu: 100, Mem: 209715200 
Jun 12 20:54:33.230: INFO: Pod for on the node: kubernetes-dashboard-1971317102-nzgtf, Cpu: 100, Mem: 104857600 
Jun 12 20:54:33.230: INFO: Node: gke-bootstrap-e2e-default-pool-5334c705-nb00, totalRequestedCpuResource: 770, cpuAllocatableMil: 2000, cpuFraction: 0.385 
Jun 12 20:54:33.230: INFO: Node: gke-bootstrap-e2e-default-pool-5334c705-nb00, totalRequestedMemResource: 962592768, memAllocatableVal: 7848955904, memFraction: 0.12263959433246932 
```

**Comment by monopole** [comment]
This test failing every other run.  Fix or disable?  Blocking 1.7?

**Comment by k82cn** [comment]
similar to #47381 , move to 1.8, but I'll still working on that.
